span_sink.go

```diff
@@ -389,3 +389,27 @@ func (ls *lightStepSpanSink) Flush() {
 	ls.serviceCount = make(map[string]int64)
 	log.WithField("total_spans", totalCount).Debug("Checkpointing flushed spans for Lightstep")
 }
+
+type blackholeSpanSink struct {
+}
+
+func NewBlackholeSpanSink() (*blackholeSpanSink, error) {
```

Comment:
Why does this return an error if it's always going to be nil? (ie, if this is intended to satisfy an interface, we should do a `var _ myInterface = ...` to self-document that here).